decidim-surveys/decidim-surveys.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "decidim-core", Decidim::Surveys.version
   s.add_dependency "decidim-forms", Decidim::Surveys.version
+  s.add_dependency "decidim-templates", Decidim::Surveys.version
```

Comment:
Hi @decidim/core! Should this dependency be moved to `decidim-forms`? 

As other modules like `decidim-meetings` or `decidim-elections` also use questionnaires from `decidim-forms` or should it be added to each module that has a dependency with `decidim-forms`?

https://github.com/decidim/decidim/blob/a179b49ed88662c1dcb0229eae61611e86ceb9a0/decidim-meetings/decidim-meetings.gemspec#L26

https://github.com/decidim/decidim/blob/f3f79b4a6bcff82e2cb4140e494936fa3f4e22bb/decidim-elections/decidim-elections.gemspec#L22

Reply:
not sure, you could used `decidim-forms` without templates if you want.

Reply:
Hi @microstudi 😄, I found this line, in `decidim-forms`, that raised an error without `decidim-templates` in `decidim-elections`:

https://github.com/decidim/decidim/blob/a179b49ed88662c1dcb0229eae61611e86ceb9a0/decidim-forms/app/views/decidim/forms/admin/questionnaires/_form.html.erb#L5

Reply:
Then I'd say that's a bug, we should check for Template to be defined. But you may also be right and just made forms depedant on Templates whether you use it or not. We'll look at it, thanks for pointing that out!

Reply:
Ok, nice feature by the way!